tests/ltp/patches/fgetxattr01.patch

```diff
@@ -0,0 +1,72 @@
+diff --git a/testcases/kernel/syscalls/fgetxattr/fgetxattr01.c b/testcases/kernel/syscalls/fgetxattr/fgetxattr01.c
```

Comment:
Add some description about the patch in .patch file. Why this change is necessary? And what is the change?